src/styles/blocks/calendar.scss

```
@@ -0,0 +1,28 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calendarWidth($cellSize, $cellShift);
+  padding: $calendarPadding;
+  gap: $cellShift;
+
+  $week:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  @each $day, $shift in $week {
+    &--start-day_#{$day} :first-child {
+      margin-left: $cellSize * $shift + $cellShift * $shift;
+    }
+  }
+
+  @for $days from 28 through 31 {
+    &--month-length_#{$days} :nth-child(n+#{$days+1}) {
```

```suggestion
    &--month-length_#{$days} :nth-child(n + #{$days + 1}) {
```
Думаю, краще робити пробіли де є чисельні вираховування, щоб код було легше читати